app/models/form_profile.rb

```diff
@@ -89,8 +88,6 @@ class FormProfile
   include Virtus.model
   include SentryLogging
 
-  EMIS_PREFILL_KEY = 'emis_prefill'
```

Comment:
If we are still using emis in the else of the conditional, do we still need this?

Reply:
`EMIS_PREFILL_KEY` is an unused variable in the repo.